api/src/main/resources/openmrs-distro.properties

```diff
@@ -0,0 +1,12 @@
+# SDK server properties file
+# import the UgandaEMR distro settings
+
+name=UgandaEMRPOC
+version=${project.parent.version}
+distro.aijar-api=${ugandaemrVersion}
+distro.aijar-api.groupId=org.openmrs.module
+omod.ugandaemrpoc=${project.parent.version}
+omod.locationbasedaccess=${locationbasedaccessVersion}
+omod.appui=${appuiVersion}
+omod.adminui=${adminuiVersion}
```

Comment:
@slubwama Do not put adminui or appui versions here - the version are maintained in aijar

Reply:
@slubwama The versions here too